.github/workflows/build_images_uv.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Image CI
+
+on:
+  pull_request:
+    paths:
+      - "lumigator/**"
+      - ".github/**"
+    branches: ["main"]
+    # synchronized is when you push new commits
+    types: ["opened", "synchronize"]
+  push:
+    branches:
+      - main
+  # required to enable manual triggers on the GH web ui
+  workflow_dispatch:
+
+env:
+  AWS_REGION: us-east-2
+
+# https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services
+permissions:
+  id-token: write
+  contents: read # This is required for actions/checkout
+
+jobs:
+  build:
+    name: Build Docker image
+    runs-on: ubuntu_2204_150gb # this is a mzai-custom image on github actions runners.
+    env:
+      # head_ref: source branch of the pull request.
+      # only set on PRs; e.g., for a PR from branch `myname/amazing_model`, will be `myname/amazing_model`
+      # ref_name: short ref name of branch triggering runs. works for pushes to a development branch.
+      # assignment works starting with the head_ref
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+    # set up a local registry
+    services:
```

Comment:
I don't think we'll need the local registry anymore

Reply:
Yeah, you're right